.github/workflows/tests.yml

```diff
@@ -51,8 +51,9 @@ jobs:
       - name: black
         uses: psf/black@stable
         with:
-          options: "--check --verbose --line-length=79 --required-version 24.1.1"
+          options: "--check --verbose --line-length=79"
           src: "./scripts"
+          version: "24.1.1"
```

Comment:
But doesn't that mean that everytime we upgrade Black, we have to change the version in two places? 🤔